spirv_msl.cpp

```diff
@@ -10682,68 +11721,541 @@ void CompilerMSL::OpCodePreprocessor::check_resource_write(uint32_t var_id)
 		uses_resource_write = true;
 }
 
-// Returns an enumeration of a SPIR-V function that needs to be output for certain Op codes.
-CompilerMSL::SPVFuncImpl CompilerMSL::OpCodePreprocessor::get_spv_func_impl(Op opcode, const uint32_t *args)
+// Storage buffer robustness
+// Fix loads from tessellation control inputs not being forwarded to the gl_in structure array
+// Fix loads from tessellation evaluation inputs not being forwarded to the stage_in structure array
+std::string CompilerMSL::access_chain_internal(uint32_t base, const uint32_t *indices, uint32_t count,
```

Comment:
Does this function have to override the entire access_chain_internal from GLSL? Consider adding just a few virtuals to deal with robustness if necessary.